native/include/mod_proxy_cluster.h

```diff
@@ -55,4 +55,32 @@ int (* proxy_node_isup)(request_rec *r, int id, int load);
 int (* proxy_host_isup)(request_rec *r, char *scheme, char *host, char *port);
 };
 typedef struct balancer_method balancer_method;
+
+/* From apr_reslist.c
```

Comment:
It is consistent between httpd 2.4.6 and httpd 2.2.26. Is there any better way how to avoid ```dereferencing pointer to incomplete type```?